api/indexer.oas2.json

```diff
@@ -692,6 +692,16 @@
           "description": "\\[spend\\] the address against which signing should be checked. If empty, the address of the current account is used. This field can be updated in any transaction by setting the RekeyTo field.",
           "type": "string",
           "x-algorand-format": "Address"
+        },
+        "created-at-round": {
+          "description": "Round during which this account first appeared in a transaction.",
+           "type": "integer",
+           "x-algorand-format": "uint64"
+        },
+        "closed-at-round": {
+          "description": "Round during which this account was most recently closed.",
+           "type": "integer",
+           "x-algorand-format": "uint64"
```

Comment:
Account:
* Create - this is the only new term, since currently the code assumes all accounts already exist (i.e. an account with no state is a valid account)
* ClosedAtRound - Closed because we use a "CloseTo" transaction to remove the state of an account.